01.py

```diff
@@ -1,19 +1,19 @@
+import sys
+
 p1 = 0
 p2 = 0
-with open('input/1.txt') as f:
-    lines = list(map(int,f.readlines()))
 
-    for i in range(0,len(lines)):
-        if i > 0:
-            if lines[i] > lines[i-1]:
-                p1 += 1
-        if i > 2:
-            v1 = sum(lines[i-3:i])
-            v2 = sum(lines[i-4:i-1])
-            if v1 > v2:
-                p2 += 1
+lines = list(map(int,sys.stdin.readlines()))
+
+for i in range(0,len(lines)):
+    if i > 0:
+        if lines[i] > lines[i-1]:
+            p1 += 1
+    if i > 2:
+        v1 = sum(lines[i-3:i])
+        v2 = sum(lines[i-4:i-1])
+        if v1 > v2:
```

Comment:
So, this logic is essentially repeating the logic of part 1. Granted, the logic is very small, but could you abstract that logic into a single function, to handle both part 1 and part 2? You will have to change your representation for part 2.